decidim-core/spec/services/decidim/open_data_exporter_spec.rb

```diff
@@ -146,7 +146,7 @@
         end
 
         it "includes the meetings data" do
-          expect(csv_data).to include(meeting.title["en"])
+          expect(csv_data).to include(translated(meeting.title).gsub("\"", "\"\""))
```

Comment:
This isn't done in develop

Reply:
Actually this comes from #10044 , I will patch to match the code you've added there.